app/templates/subsidy/detail/step/detail.hbs

```diff
@@ -180,37 +133,69 @@
               </AuAlert>
             </div>
           {{else}}
-            {{! GENERAL ISSUES (data is broken) }}
-            {{#if this.isActiveStep}}
+            {{#if this.submittablePeriodNeedsToStart}}
               <div>
                 <AuAlert
+                  class="au-u-margin-bottom-small"
                   @icon="info-circle"
-                  @title="Algemene fout"
-                  @skin="error"
+                  @skin="info"
                   @size="small"
-                  class="au-u-margin-bottom-small"
+                  @title="Deze stap kan u nog niet indienen."
                 >
-                  <p>Indien dit blijft gebeuren, neem contact op met
-                    <a
-                      href="mailto:LoketLokaalBestuur@vlaanderen.be"
-                      class="au-c-link"
-                    >
-                      LoketLokaalBestuur@vlaanderen.be
-                    </a>.
-                  </p>
+                  <AuContent>
+                    <p>U kan pas vanaf een later moment indienen.</p>
+
+                    <p>
+                      Meer informatie over de subsidieaanvragen vindt u
+                      <AuLinkExternal
+                        href="https://www.vlaanderen.be/lokaal-bestuur/loket-voor-lokale-besturen/subsidiebeheer"
+                      >
+                        hier
+                      </AuLinkExternal>.<br />
+                      Indien u een fout heeft ontdekt, of een vraag heeft, neem
+                      contact op met
+                      <a
+                        class="au-c-link"
+                        href="mailto:LoketLokaalBestuur@vlaanderen.be"
+                      >
+                        LoketLokaalBestuur@vlaanderen.be
+                      </a>.
+                    </p>
+                  </AuContent>
                 </AuAlert>
+                {{! Buttons not needed in readonly detail view }}
```

Comment:
Why not simply remove this instead of commenting it?

Reply:
Removed it now, we wanted to keep it around for a little bit in case we ended up needing it.